src/utils/misc.py

```diff
@@ -6,7 +6,7 @@
 
 def get_datadir(user_type):
     if user_type == "user":
-        return os.path.join(GLib.get_user_data_dir(), PACKAGE)
+        return os.path.join(GLib.get_user_data_dir())
     elif user_type == "system":
         return os.path.join(DATADIR, PACKAGE)
```

Comment:
Should we change it for the system dir too? Probably have to change the installation of the sounds as well.

Reply:
I'm not sure we really want to put them in `/app/share` since they could conflict with anything else that we need to build as part of this package.

Reply:
what other directory would you suggest?

Reply:
I would leave the system directory as it was.